.github/workflows/get-heroku-app-commit-index.yml

```diff
@@ -0,0 +1,69 @@
+name: Get Heroku slug commit and resource index
+
+on:
+  workflow_call:
+    inputs:
+      app_name:
+        description: Heroku app name
+        type: string
+        required: true
+    outputs:
+      commit_hash:
+        description: The commit hash of the latest Heroku slug for the app.
+        value: ${{ jobs.heroku_app_info.outputs.commit_hash }}
+      resource_index:
+        description: The RESOURCE_INDEX extracted from the env/production/config.json
+        value: ${{ jobs.heroku_app_info.outputs.resource_index }}
+
+defaults:
+  run:
+    # This is the same as GitHub Action's `bash` keyword as of 20 June 2023:
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
+    #
+    # Completely spelling it out here so that GitHub can't change it out from under us
+    # and we don't have to refer to the docs to know the expected behavior.
+    shell: bash --noprofile --norc -eo pipefail {0}
+
+jobs:
+  heroku_app_info:
+    runs-on: ubuntu-latest
+    env:
+      HEROKU_TOKEN: ${{ secrets.HEROKU_TOKEN_READ_PROTECTED }}
+      HEROKU_PIPELINE: 38f67fc7-d93c-40c6-a182-501da2f89d9d
+      APP_NAME: ${{ inputs.app_name }}
+    outputs:
+      commit_hash: ${{ steps.commit_hash.outputs.commit_hash }}
+      resource_index: ${{ steps.resource_index.outputs.resource_index }}
+    steps:
+      - name: Get latest slug ID for ${{ env.APP_NAME }}
+        run: |
+          slug_id=$(
+            curl -n https://api.heroku.com/pipelines/"$HEROKU_PIPELINE"/latest-releases \
+              --fail --silent --show-error \
+              -H "Accept: application/vnd.heroku+json; version=3" \
+              -H "Authorization: Bearer $HEROKU_TOKEN" \
+              | jq -r '.[] | select(.app | .name == env.APP_NAME) | .slug.id')
+
+          echo "SLUG_ID=$slug_id" | tee -a "$GITHUB_ENV"
+
+      - id: commit_hash
+        name: Get latest commit hash for ${{ env.app_name }}
```

Comment:
Ah, that would be a typo! Thanks for catching it.

Reply:
No longer relevant since I moved this piece into a separate shell script in https://github.com/nextstrain/nextstrain.org/pull/841/commits/a90c1c35da5265a1680ae513bcb701615feaba94